vivisect/tests/testmacho.py

```diff
@@ -0,0 +1,241 @@
+import logging
+import unittest
+
+import vstruct.defs.macho as vs_macho
+import vstruct.defs.macho.const as vsm_const
+
+import vivisect.tests.utils as utils
+import vivisect.tests.helpers as helpers
+import vivisect.parsers.macho as v_p_macho
+
+logger = logging.getLogger(__name__)
+
+
+class MachOTests(utils.VivTest):
+    @classmethod
+    def setUpClass(cls):
+        super(MachOTests, cls).setUpClass()
+        byts = helpers.getTestBytes('osx', 'amd64', 'safaridriver.bin')
+        cls.safari = vs_macho.mach_o()
+        cls.safari.vsParse(byts)
+        #cls.safari_vw = helpers.getTestWorkspace('osx', 'amd64', 'safaridriver.bin')
```

Comment:
what's going on here?  are we adding the workspace for testing?  or should this be deleted?

Reply:
Need to remove. I was originally planning on fully plumbing through full analysis for macho files, but I figured that was better saved for a Mach-O part 2 because there's most nuances there to deal with.

Reply:
replace it with a # TODO :)
let's knock this PR out, then plumb the rest through later, now that we'll have unittests and scaffolding around what works.